srcdoc/configure/server_timing_directives.mt

```diff
@@ -0,0 +1,121 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Server Timing Directives")->(sub {
+
+<p>
+The server timing handler turns on the <a href="https://www.w3.org/TR/server-timing/">Server Timing</a> feature.
+This document describes the configuration directives of the handler.
+</p>
+
+? $ctx->{directive_list}->()->(sub {
+
+<?
+my $access_log_ext_link = sub {
+    my $name = shift;
+    return $_mt->filter(sub {
+        "<a href=\"configure/access_log_directives.html#$name\"><code>$name</code></a>"
+    })->(sub {});
+};
+$ctx->{directive}->(
+    name    => "server-timing",
+    levels  => [ qw(global host path extension) ],
+    default => 'OFF',
+    since   => '2.3',
+    desc    => <<'EOT',
+A flag indicating how <code>server-timing</code> header and trailer should be served.
+EOT
+)->(sub {
+?>
+
+<p>
+If the argument is <code>ON</code>, H2O adds the <code>server-timing</code> header and trailer (if possible) to the response. The header is always added, but the trailer isn't if all of the following conditions are met.
```

Comment:
In other references to the possible values (e.g. L38, L42), you use the word `value`. Would it help to keep the narrative consistent?